numpy/core/src/common/npy_cpu_features.c.src

```diff
@@ -93,24 +96,10 @@ npy__cpu_cpuid(int reg[4], int func_id)
 #elif defined(__INTEL_COMPILER)
     __cpuid(reg, func_id);
 #elif defined(__GNUC__) || defined(__clang__)
-    #if defined(NPY_CPU_X86) && defined(__PIC__)
-        // %ebx may be the PIC register
-        __asm__("xchg{l}\t{%%}ebx, %1\n\t"
-                "cpuid\n\t"
-                "xchg{l}\t{%%}ebx, %1\n\t"
-                : "=a" (reg[0]), "=r" (reg[1]), "=c" (reg[2]),
-                  "=d" (reg[3])
-                : "a" (func_id), "c" (0)
-        );
-    #else
-        __asm__("cpuid\n\t"
-                : "=a" (reg[0]), "=b" (reg[1]), "=c" (reg[2]),
-                  "=d" (reg[3])
-                : "a" (func_id), "c" (0)
-        );
-    #endif
+    #include <cpuid.h>
```

Comment:
Old versions of clang-3 and gcc-4 don't support it, not sure why you want to use `__cpuid_count` instead?

Reply:
I got this information from [Google Cpu Detect Project](https://github.com/google/cpu_features/blob/master/src/cpuinfo_x86.c), But like you said, The compiler version needs to higher than clang-3 and gcc-4, I will revert it.